### PR TITLE
Oracles

### PR DESCRIPTION
**Pull request by alexroan**

### CODE REVIEW

File: contracts/USM.sol [hunk 1]
```diff
@@ -2,7 +2,7 @@ pragma solidity ^0.6.7;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
-import "./IOracle.sol";
+import "./oracles/IOracle.sol";
```
Comment (alexroan) [inline contracts/USM.sol:5]: Moved all oracle contracts to their own folder

File: contracts/oracles/ChainlinkOracle.sol [hunk 2]
```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.6.7;
```
Comment (alexroan) [inline contracts/oracles/ChainlinkOracle.sol:1]: Created the Chainlink oracle

File: contracts/oracles/TestOracle.sol [hunk 3]
```diff
@@ -4,7 +4,7 @@ import "./IOracle.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
-contract USDOracle is IOracle, Ownable{
+contract TestOracle is IOracle, Ownable{
```
Comment (alexroan) [inline contracts/oracles/TestOracle.sol:7]: Renamed this to TestOracle as it only serves as input for our tests.

File: build/contracts/Address.json [hunk 4]
```diff
@@ -1,5834 +0,0 @@
-{
```
Comment (alexroan) [inline build/contracts/Address.json:1]: Removed previously tracked `build` files

**Review by albertocuestacanada — APPROVED** [review]
All good, keep them coming!

**Pull request merged**